internal/services/signalr/web_pubsub_socketio_resource.go

```diff
@@ -0,0 +1,588 @@
+	Location                         string                                     `tfschema:"location"`
+	Sku                              string                                     `tfschema:"sku"`
+	AADAuthEnabled                   bool                                       `tfschema:"aad_auth_enabled"`
+	Capacity                         int64                                      `tfschema:"capacity"`
+	Identity                         []identity.ModelSystemAssignedUserAssigned `tfschema:"identity"`
+	LiveTraceEnabled                 bool                                       `tfschema:"live_trace_enabled"`
+	LiveTraceConnectivityLogsEnabled bool                                       `tfschema:"live_trace_connectivity_logs_enabled"`
+	LiveTraceHttpRequestLogsEnabled  bool                                       `tfschema:"live_trace_http_request_logs_enabled"`
+	LiveTraceMessagingLogsEnabled    bool                                       `tfschema:"live_trace_messaging_logs_enabled"`
+	LocalAuthEnabled                 bool                                       `tfschema:"local_auth_enabled"`
+	PublicNetworkAccess              string                                     `tfschema:"public_network_access"`
+	ServiceMode                      string                                     `tfschema:"service_mode"`
+	Tags                             map[string]string                          `tfschema:"tags"`
+	TlsClientCertEnabled             bool                                       `tfschema:"tls_client_cert_enabled"`
+	ExternalIP                       string                                     `tfschema:"external_ip"`
+	HostName                         string                                     `tfschema:"hostname"`
+	PrimaryAccessKey                 string                                     `tfschema:"primary_access_key"`
+	PrimaryConnectionString          string                                     `tfschema:"primary_connection_string"`
+	PublicPort                       int64                                      `tfschema:"public_port"`
+	ServerPort                       int64                                      `tfschema:"server_port"`
+	SecondaryAccessKey               string                                     `tfschema:"secondary_access_key"`
+	SecondaryConnectionString        string                                     `tfschema:"secondary_connection_string"`
+}
+
+var (
+	_ sdk.ResourceWithUpdate        = WebPubSubSocketIOResource{}
+	_ sdk.ResourceWithCustomizeDiff = WebPubSubSocketIOResource{}
+)
+
+func (w WebPubSubSocketIOResource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:         pluginsdk.TypeString,
+			Required:     true,
+			ForceNew:     true,
+			ValidateFunc: validate.WebPubSubName(),
+		},
+
+		"resource_group_name": commonschema.ResourceGroupName(),
+
+		"location": commonschema.Location(),
+
+		"sku": {
```

Comment:
Going back to our sku/capacity internal discussion, do we want to put these under a `sku` block?